src/autoscaler/metricscollector/cmd/metricscollector/metricscollector_suite_test.go

```diff
@@ -205,6 +231,20 @@ func (mc *MetricsCollectorRunner) Start() {
 	mc.Session = mcSession
 }
 
+func (mc *MetricsCollectorRunner) StartWithoutCheck() {
```

Comment:
Since Start() above is already having a check mc.startCheck != "", so can reuse it to do what StartWithoutCheck() is doing.